src/app/modules/monitoring/group-monitoring/enrollment/enrollment.component.html

```diff
@@ -14,14 +17,14 @@
             <div *ngIf="learnerEnrollment.enrollment" fxLayoutAlign=" center" fxLayoutGap="15px">
                 <small>
                     Status: {{learnerEnrollment.enrollment.status}}<br>
```

Comment:
Will lead to inconsistent view if only one learner is unenrolled. Because unenroll does not return enrollment, but bulk unenroll returns list of enrollments. Not sure what is desired in this situation.
![image](https://github.com/Clean-CaDET/platform-tutor-ui-web/assets/18446751/b945ddc0-d484-4efd-8a0b-e447f63744d4)

Reply:
Good catch. I came across this issue, but decided to ignore it for now as it is not significant. Since the current logic makes a DELETE request and we are in reality updating the enrollment, the HTTP method should be changed and the hidden enrollment can be returned to update everything.

I will leave it as it is for now and create a clickup card https://app.clickup.com/t/861mwa5bm